modules/gcp_bigquery_table/README.md

```diff
@@ -1,11 +1,11 @@
-This module is using for create BigQuery table
+This module is using for create BigQuery View
```

Comment:
Is this correct? `google_bigquery_table` is a BigQuery View?

Reply:
correct, view = table that flag as view

Reply:
done